.github/workflows/codeql.yml

```diff
@@ -0,0 +1,39 @@
+name: 'CodeQL'
+
+on:
+  push:
+    branches: ['main']
+  pull_request:
+    branches: ['main']
+  schedule:
```

Comment:
why schedule?

Reply:
Literally just copied and pasted this from #1599. Removed 15c7043